todo_app/app.py

```diff
@@ -1,15 +1,22 @@
-from flask import Flask
+from flask import Flask, render_template, redirect, url_for, request
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
+@app.route('/', methods=['GET'])
 def index():
-    return 'Hello World!'
-
+    items3 = get_items()
+    return render_template('index.html',items=items3)
 
+@app.route('/add-todo', methods=['GET','POST'])
```

Comment:
Now this is working, we could limit this to just POST requests, since that's all we're using it for currently